README.md

```diff
@@ -183,4 +347,4 @@ The latest version of Synopsys Bridge will be downloaded by default if user does
 
 ## Setting Up Synopsys Bridge Manually
```

Comment:
The latest version of Synopsys Bridge will be downloaded by default if user doesn't provide the specific released version  in the `pipeline parameter`/ **Global UI** or the installed version is not the latest version.